app/controllers/course/condition/achievements_controller.rb

```diff
@@ -0,0 +1,43 @@
+class Course::Condition::AchievementsController < Course::ConditionsController
+  load_resource instance_name: :achievement_condition, class: Course::Condition::Achievement.name,
+                param_method: :achievement_condition_params
+
+  def new
+  end
+
+  def edit
+  end
+
+  def create
```

Comment:
Are you sure this is our convention? The [official guide](http://guides.rubyonrails.org/getting_started.html#showing-articles) suggests this order: `index, show, new, edit, create, update and destroy`. Our existing controller for `Achievement` model also uses this order.